doc/Security.xml

```diff
@@ -890,7 +938,7 @@ Added certificate-based client authentication</revremark>
                     <para>False</para>
                   </entry>
                   <entry>
-                    <para>If true, a TLS server shall only allow TLS clients to connect that present a client certificate containing the Require WWW client auth extended key usage extension as specified in RFC 5280, Sect. 4.2.1.12.</para>
+                    <para>If true, a TLS server shall only allow TLS clients to connect that present a client certificate containing the Require WWW client auth extended key usage extension as specified in [RFC 5280], Sect. 4.2.1.12.</para>
```

Comment:
IEC don't put referenced document inside "[ ]". See section 10 (e.g. 10.4) in https://www.iso.org/sites/directives/current/part2/index.xhtml

Reply:
@cappentz shall I remove the usage of [] in the whole document? It is used a lot

Reply:
@HansBusch do you agree removing [ ] for references?